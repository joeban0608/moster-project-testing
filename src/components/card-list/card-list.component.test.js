import { shallow, mount, render } from "enzyme";
import React from "react";
import CardList from "./card-list.component";

const mockMonsters = [
  {
    id: 10,
    name: "Clementina DuBuque",
    email: "Rey.Padberg@karina.biz",
  },
  {
    id: 11,
    name: "Joe",
    email: "Joe@karina.biz",
  },
];

it("is render CardList component", () => {
  // expect(shallow(<Card monster={mockMonster} />).length).toEqual(1)
  expect(shallow(<CardList monsters={mockMonsters} />)).toMatchSnapshot();
});
