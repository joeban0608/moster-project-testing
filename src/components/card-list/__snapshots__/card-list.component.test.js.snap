// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is render CardList component 1`] = `
<div
  className="card-list"
>
  <Card
    key="10"
    monster={
      Object {
        "email": "Rey.Padberg@karina.biz",
        "id": 10,
        "name": "Clementina DuBuque",
      }
    }
  />
  <Card
    key="11"
    monster={
      Object {
        "email": "Joe@karina.biz",
        "id": 11,
        "name": "Joe",
      }
    }
  />
</div>
`;
